### PR TITLE
Add HLS to asset params

### PR DESCRIPTION
**Pull request by grod220**
Max LTV and Liquidation threshold now have high-levered versions. Adding to asset params contract so Rover can use.

### CODE REVIEW

File: contracts/params/src/execute.rs [hunk 1]
```diff
@@ -112,3 +112,17 @@ pub fn assert_lqt_gte_max_ltv(
     }
     Ok(())
 }
+
+pub fn assert_hls_lqt_gte_max_ltv(
```
Comment (piobab) [inline contracts/params/src/execute.rs:116]: error for `if liq_threshold <= max_ltv` so assert function name just without `e` `assert_hls_lqt_gt_max_ltv`

File: contracts/params/src/types.rs [hunk 2]
```diff
@@ -2,15 +2,15 @@ use cosmwasm_schema::cw_serde;
...
+use crate::execute::{assert_hls_lqt_gte_max_ltv, assert_lqt_gte_max_ltv};
 
 #[cw_serde]
 pub struct RoverPermissions {
     pub whitelisted: bool,
 }
 
 #[cw_serde]
-pub struct RedBankSettings {
+pub struct RedBankPermissions {
```
Comment (piobab) [inline contracts/params/src/types.rs:13]: Why change?

Comment (piobab) [reply contracts/params/src/types.rs:13]: I think we decided to name them `Settings`

File: contracts/params/src/types.rs [hunk 3]
```diff
@@ -19,12 +19,19 @@ pub struct RedBankSettings {
...
+#[cw_serde]
+pub struct HighLeverageStrategyParams {
+    pub max_loan_to_value: Decimal,
+    pub liquidation_threshold: Decimal,
 }
 
 #[cw_serde]
 pub struct AssetParams {
     pub permissions: AssetPermissions,
+    pub hls: HighLeverageStrategyParams,
```
Comment (piobab) [inline contracts/params/src/types.rs:34]: hmm maybe include in Rover settings?

File: contracts/params/tests/test_asset_validation.rs [hunk 4]
```diff
@@ -120,3 +120,73 @@ fn liq_threshold_gte_max_ltv() {
...
+        Validation(InvalidParam {
+            param_name: "hls_liquidation_threshold".to_string(),
+            invalid_value: "1.1235".to_string(),
+            predicate: "<= 1".to_string(),
+        }),
+    );
+}
+
+#[test]
+fn hls_liq_threshold_gte_hls_max_ltv() {
```
Comment (piobab) [inline contracts/params/tests/test_asset_validation.rs:171]: `hls_liq_threshold_gt_hls_max_ltv`

File: contracts/params/src/types.rs (hunk 2 above)
Comment (piobab) [reply contracts/params/src/types.rs:13]: I think both should be settings. Probably we missed it during renaming.

**Review by piobab — APPROVED** [review]

**Review by brimigs — APPROVED** [review]
👍

**Pull request merged**